docs/user/gcp/install_xpn.md

```diff
@@ -0,0 +1,167 @@
+# Install: Installer provisioned infrastructure using Shared VPC (XPN)
+The steps for performing a installer provisioned infrastructure installation
+using Shared VPC (XPN) networking resources.
+
+## Create an base install config
+Create the `install-config.yaml` manually using the documentation
+references/examples as you would for a normal IPI Installation.
+
+### Configure passthrough credentialsMode
+The cluster will need permissions in the Host Project in order to function
+properly. This will be accomplished by configuring the service account used by
+the installer to have these permissions and configuring the credentialsMode to
+be passthrough. This configures the cluster to use the same service account
+internally after installation. The exact permissions vary depending on choices
+selected throughout this document as specified below.
+
+```
+credentialsMode: Passthrough
+```
+
+### Add pre-exsiting XPN network.
+The following parameters are necessary to configure the installer to use
+pre-exsting network resources, and specifcy that they exist in the
+networkProjectID project, also known as the "Host Project".
+
+```
+platform:
+  gcp:
+    computeSubnet: <compute-subnet>
+    controlPlaneSubnet: <control-plane-subnet>
+    network: <network>
+    networkProjectID: <networkProjectID>
+```
+
+The installer service account requires the following permissions on the network
+in the Host Project in order to enable the cluster to use the network resources.
+
+- Compute Network User
```

Comment:
nit: Should we link the information about the role here? (https://cloud.google.com/compute/docs/access/iam#compute.networkUser)